sklearn/svm/_classes.py

```diff
@@ -362,9 +367,11 @@ class LinearSVR(RegressorMixin, LinearModel):
         To lessen the effect of regularization on synthetic feature weight
         (and therefore on the intercept) intercept_scaling has to be increased.
 
-    dual : bool, default=True
+    dual : {"auto", False, True}, default= "auto"
         Select the algorithm to either solve the dual or primal
-        optimization problem. Prefer dual=False when n_samples > n_features.
+        optimization problem.
+        Sets dual = False when n_samples > n_features if dual is "auto"
+        and dual = True otherwise.
```

Comment:
```suggestion
        `dual="auto"` is equivalent to `dual=False` when
        `n_samples > n_features` and `dual=True` otherwise.

        .. versionchanged:: 1.4
           The default value will change from `True` to `"auto"` in 1.4.
```